client/src/components/Queue/Track.jsx

```diff
@@ -1,48 +1,53 @@
-import React, { useState, useEffect } from 'react';
-import styled from 'styled-components';
-import axios from 'axios';
-import { FaThumbsUp, FaThumbsDown } from 'react-icons/fa';
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import axios from "axios";
+import { FaThumbsUp, FaThumbsDown } from "react-icons/fa";
```

Comment:
Is there a reason why you switched the single quotes here to double quotes? I image eslint to not be too happy with this (no judging from me; I hate eslint 😠 )